tests/unit/IOUUtilsTest.js

```diff
@@ -11,35 +12,29 @@ let reportActions;
 const ownerEmail = 'owner@iou.com';
 const managerEmail = 'manager@iou.com';
 
-function createIOUReportAction(type, amount, currency, {IOUTransactionID, isOnline} = {}) {
+function createIOUReportAction(type, amount, currency, isOffline = false, IOUTransactionID = NumberUtils.rand64()) {
     const moneyRequestAction = ReportUtils.buildOptimisticIOUReportAction(
         type,
         amount,
         currency,
         'Test comment',
         [managerEmail],
-        '',
         IOUTransactionID,
+        '',
         iouReport.reportID,
     );
 
-    // Default is to create requests offline, if this is specified then we need to remove the pendingAction
-    moneyRequestAction.pendingAction = isOnline ? null : 'add';
+    // Default is to create requests online, if `isOffline` is specified then we need to remove the pendingAction
```

Comment:
```suggestion
    // Default is to create requests online, if `isOffline` is not specified then we need to remove the pendingAction
```
Sorry but you missed that tiny change